src/Eccube/Application.php

```diff
@@ -602,10 +606,10 @@ public function loadPlugin()
         // config.yml/event.ymlの定義に沿ってインスタンスの生成を行い, イベント設定を行う.
         foreach ($finder as $dir) {
             //config.ymlのないディレクトリは無視する
-            if (!file_exists($dir->getRealPath() . '/config.yml')) {
+            if (!file_exists($dir->getRealPath().'/config.yml')) {
                 continue;
             }
-            $config = Yaml::parse($dir->getRealPath() . '/config.yml');
+            $config = Yaml::parse($dir->getRealPath().'/config.yml');
```

Comment:
__[phpmd]__
* Avoid using static access to class '\Symfony\Component\Yaml\Yaml' in method 'loadPlugin'.